docs/product/issues/issue-details/index.mdx

```diff
@@ -88,23 +107,34 @@ You can set your own <PlatformLink to="/enriching-events/breadcrumbs/">breadcrum
 
 If you’ve enabled [Session Replay](/product/session-replay/), you’ll see a replay preview under Breadcrumbs if there’s one associated with the event you’re viewing. Replays can be associated with both frontend and [backend errors](/product/session-replay/getting-started#replays-for-backend-errors) (as long as distrubted tracing is set up). Clicking on the replay preview will lead you to the [Replay Details](/product/session-replay/replay-details/) page.
 
-## Trace Details
```

Comment:
Opted to remove this section since it describes one type of context when there are endless others. At best, it links out to distributed tracing (something the #Trace Navigator section already does) and mentions that there's a link.

Instead I've added the contexts section to replace it.